data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,88 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Freighters to Zug for an important project. Expect pirate raids."
+	source
+		government "Free Worlds"
+		near Zubeneschamali 2 4
+	destination "Zug"
+	to offer
+		has "event: war begins"
+		has "FW Syndicate Diplomacy 1D: done"
+		not "fw bloodsea done"
+	on offer
+		conversation
+			`While returning from a walk around the spaceport town of <origin>, you see the quartermaster of the spaceport inspecting the <ship>. He notices your arrival and comes up to greet you.`
```

Comment:
We just go away from "the `<ship>`" in #7718, so let's avoid it here too. I'd also drop "town" because we could be at a large spaceport, and "town" would seem like a weird descriptor to me in that case.
```suggestion
			`While returning from a walk around the spaceport of <origin>, you see the quartermaster of the spaceport inspecting your ship. He notices your arrival and comes up to greet you.`
```